### ISSUE TITLE
How to recreate POST request to already exist special page via official API?

### ISSUE THREAD

**Issue by rojenzaman** [melmothx/amusewiki#361]
Hi,

 I was searching API documentation and  I found  only  this documentation: [Integration with external sites](https://amusewiki.org/library/integration-with-external-sites)

This document explains only how to create a post request when creating new article, so its no tell about other post requests.

All I need is how to edit the special page via API. Can you explain?

Sample special page:
https://amuse-site.tld/action/special/edit/SOMEONEPAGE
https://amuse-site.tld/special/SOMEONEPAGE

### like this

```bash

curl -F __auth_pass="passwd" \
       -F __auth_user="user" \
       -F title="SOMEONE SPECIAL PAGE" \
       -F textbody='Updated Text' \
       https://amuse-site.tld/special/SOMEONEPAGE   
```

**Comment by melmothx** [comment]
Rojen Zaman <notifications@github.com> writes:

> Hi,
>
>  I was searching API documentation and I found only this
>  documentation: [Integration with external
>  sites](https://amusewiki.org/library/integration-with-external-sites)
>
> This document explains only how to create a post request when creating
> an article, so its no tell about other post requests.
>
> All I need is how to edit the special page via API. Can you explain?
>
> Sample special page:
> https://amuse-site.tld/action/special/edit/SOMEONEPAGE
> https://amuse-site.tld/special/SOMEONEPAGE

Hi,

The only "official" API at the moment is for the article creation.

I think that adding special creation/editing and regular editing
shouldn't be a big deal, though, but will need a bit of time.

I will ask for feedback will be ready. Sounds good?

-- 
Marco

**Comment by rojenzaman** [comment]
I got it, thanks for your response.

**Comment by melmothx** [comment]
This is done and released on 2.509.

example call:

```
curl -F __auth_user="user" \
     -F __auth_pass="password" \
     -F message="Commit message" \
     -F 'body=<test.muse' https://site.amusewiki.org/remote/edit/library/test
```

Instead of `library/test` you can have `special/test` or whatever.

Documentation will be updated accordingly.

**Comment by rojenzaman** [comment]
> This is done and released on 2.509.
> 
> example call:
> 
> ```
> curl -F __auth_user="user" \
>      -F __auth_pass="password" \
>      -F message="Commit message" \
>      -F 'body=<test.muse' https://site.amusewiki.org/remote/edit/library/test
> ```
> 
> Instead of `library/test` you can have `special/test` or whatever.
> 
> Documentation will be updated accordingly.

Great work, thank you!
